.buildkite/hooks/post-checkout

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+mise install
```

Comment:
The build is currently failing on the absence of this. If we install [Mise](https://mise.jdx.dev/installing-mise.html) on the workers, everything _should_ just work. 🤞🏻